portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,12 +15,30 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private final List<String> COMMENTS = Arrays.asList("test", "json", "response");
+  private final List<String> comments = new ArrayList<>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    System.out.println("GET");
```

Comment:
Logging output would contain more useful details when using logging framework (e.g. java.util.logging):

- Built in JDK [java.util.logging] (https://docs.oracle.com/en/java/javase/11/docs/api/java.logging/java/util/logging/package-summary.html) or
- [Google's logger](https://g3doc.corp.google.com/monitoring/logging/g3doc/java_debug_logging/java-logging-best-practices.md?cl=head)

Also makes sense to add logging into other places.